src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+/**
+ * Loads the task list from a file in the hard disk and updates any changes in the
+ * task list in the same file
+ */
+public class Storage {
+
+    private final String filename;
+
+    public Storage(String filename) {
+        this.filename = filename;
+    }
+
+    /**
+     * Loads list of tasks previously saved in hard disk
+     * If no previous file exists, create a new file
+     * @return ArrayList of tasks that were loaded from hard disk
+     * @throws IOException
+     */
```

Comment:
There could be an empty line between description and parameter section as suggested here: https://se-education.org/guides/conventions/java/intermediate.html#comments
Same for other JavaDoc comments. 😃